riven/src/models.rs

```diff
@@ -3069,6 +3066,10 @@ pub mod tft_match_v1 {
         pub players_eliminated: i32,
         #[serde(rename = "puuid")]
         pub puuid: String,
+        #[serde(rename = "riotIdGameName")]
+        pub riot_id_game_name: String,
+        #[serde(rename = "riotIdTagline")]
+        pub riot_id_tagline: String,
```

Comment:
Primarily created this PR because Riot finally added game name & tag line to the TFT match API 🥴